src/Controls/src/Core/ListView.cs

```diff
@@ -458,6 +458,8 @@ protected override void UnhookContent(Cell content)
 			if (content == null || !_logicalChildren.Contains(content))
```

Comment:
Should `!_logicalChildren.Contains(content)` be removed? Otherwise it is still doing two lookups right?

```suggestion
			if (content == null)
```